app/overrides/add_spree_premade_cart_admin_link.rb

```diff
@@ -4,7 +4,7 @@
   virtual_path: 'spree/admin/shared/_configuration_menu',
   name: 'add_premade_carts_admin_menu_link',
   insert_bottom: "[data-hook='admin_configurations_sidebar_menu']",
-  text: "<%= configurations_sidebar_menu_item 'Premade Carts', admin_premade_carts_path %>"
+  text: "<%= configurations_sidebar_menu_item 'Premade Carts', admin_premade_carts_path %>",
```

Comment:
@jacobherrington were you able to reliable get this to work via `menu_items`?

Reply:
@fastjames are you referring to Spree::Backend::Config::MenuItem or our theme facing Menu Items?

I can get it working, just tests get a little weird.

Reply:
With our followup work this morning, I think we have a way to do this that removes deface without introducing bad test practices.

@LukeSettle deface has shown itself to be unreliable (as in menu items disappearing) so if we can eliminate the need for it, so much the better.